<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="numwell">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер скважины:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_numwell">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>26</y>
     <width>190</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="scheme">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>74</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Схема добавления:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_scheme">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>76</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nports">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>130</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nports">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>127</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество портов</string>
   </property>
  </widget>
  <widget class="QLabel" name="lenwing">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>180</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Длина крыльев закреплённой трещины ГРП:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lenwingleft">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>190</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lenwingright">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>190</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="para_Lleft">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Левая</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="param_Lright">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Правая</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="translatability">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Проницаемость проппантной пачки  kf, Д:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_translatability">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>272</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cracopening">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Раскрытие w, мм:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_crackopening">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>332</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
